### DISCUSSION TITLE
API support "Get announcements" (Split info)

### DISCUSSION THREAD

**Discussion by Sonny8Huang** [Petersoj/alpaca-java#128] [Q&A] [answered]
Will or does API support Alpaca End Point (/v2/corporate_actions/announcements) , specificly for stock "Split" info?

**Comment by Petersoj** [comment, marked as answer]
That API is currently unsupported by this library. I intend to do a massive update of this library so that it's up-to-date with Alpaca's API as well as has support for the Broker API! I intend to begin this update December of this year. Feel free to open a draft PR and start the work for it yourself though!
